src/goose.rs

```diff
@@ -356,11 +356,26 @@ pub enum GooseTaskError {
     /// is available in `.method`.
     InvalidMethod { method: Method },
 }
+impl GooseTaskError {
+    fn describe(&self) -> &str {
+        match *self {
+            GooseTaskError::Reqwest(_) => "reqwest::Error",
+            GooseTaskError::Url(_) => "url::ParseError",
```

Comment:
This is okay, but I would rather that we pass through information from child errors still, e.g. with `:`.

Reply:
the proposed implementation didn't work, I tried an alternative